aspnetcore/fundamentals/url-rewriting/sample/URLRewritingSample.csproj

```diff
@@ -2,9 +2,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>net451;netcoreapp1.1</TargetFrameworks>
+    <RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
```

Comment:
That shouldn't be necessary for RTM.
It's possible to work around the packages missing from NuGet by adding https://dotnet.myget.org/F/dotnet-core/api/v3/index.json to a local NuGet.Config file